.github/workflows/maven.yml

```diff
@@ -39,14 +39,14 @@ jobs:
           mvn -version
 
       - name: Build and test (headless with xvfb in Linux)
-        run: xvfb-run mvn clean verify -PuseJenkinsSnapshots -PbuildP2Repository
+        run: xvfb-run mvn clean verify -PuseJenkinsSnapshots
         if: matrix.os == 'ubuntu-latest'
-        working-directory: org.eclipse.xtext.p2.releng
+        working-directory: org.eclipse.xtext.full.releng
 
       - name: Build and test (in other OSes)
         run: mvn clean verify -PuseJenkinsSnapshots
```

Comment:
is the useJenkinsSnapshots profile still used?

Reply:
It's still used for this:

```
${JENKINS_URL}/job/lsp4j/lastStableBuild/artifact/build/maven-repository
```

Reply:
have to think about that